src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -1,37 +1,171 @@
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.PlayerInventory;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.bukkit.inventory.meta.PotionMeta;
 import org.bukkit.plugin.java.JavaPlugin;
+import org.bukkit.potion.PotionData;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
 
 import javax.validation.constraints.NotNull;
 
-public class PluginTemplate extends JavaPlugin {
+import java.util.*;
+import java.util.List;
+import java.util.logging.Logger;
+import com.sun.org.apache.bcel.internal.generic.InstructionConstants;
+import org.bukkit.potion.PotionType;
+
+import java.util.HashMap;
+import java.util.*;
+import java.util.logging.Logger;
 
+public class PluginTemplate extends JavaPlugin {
+    private static final String ARROW_TYPE_TNT = "tnt";
     private final String PLUGIN_NAME = "PluginTemplate";
 
-    public void onEnable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Started ");
+    public void onEnable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Started ");
     }
 
-    public void onReload(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Reload");
+    public void onReload() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Reload");
     }
 
-    public void onDisable(){
-        getLogger().info("["+ PLUGIN_NAME +"] Server Stopping");
+    public void onDisable() {
+        getLogger().info("[" + PLUGIN_NAME + "] Server Stopping");
     }
 
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
-        if (label.equalsIgnoreCase("simple")){
-            if (sender instanceof Player){
-                Player me = (Player)sender;
-                me.sendMessage("Plugin Working");
+        String simple="simple";
+        if (label.equalsIgnoreCase(simple)) {
+            if (sender instanceof Player) {
+                Player me = (Player) sender;
+                String message = "Plugin Working";
+                me.sendMessage(message);
                 return true;
             }
+            if (label.equalsIgnoreCase("hawkeye")) {
+                if (args[0].equalsIgnoreCase(ARROW_TYPE_TNT)) {
+                    if (sender instanceof Player) {
+                        Player me = (Player) sender;
+                        PlayerInventory inventory = me.getInventory();
+                        for (int i = 0; i < inventory.getSize(); i++) {
+                            ItemStack item = inventory.getItem(i);
+                            if (item == null) {
+                                continue;
+                            }
+                            if (isArrow(item) == true) {
+                                item.setType(Material.TIPPED_ARROW);
+                                item.setAmount(item.getAmount());
+
+                                String potion = "CONDUIT_POWER";
```

Comment:
Issue #2. Have you used Enums before, if so then this should be one, if not let me know